Description: relax dependencies
 "image" in Debian doesn't have features "avif" and "exr", so disable them.
 image:ff is new in image 0.25
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@
 path = "src/lib.rs"
 
 [dependencies.anyhow]
-version = "1.0.89"
+version = "1.0"
 
 [dependencies.audio_checker]
 version = "0.1"
@@ -74,22 +74,22 @@
 version = "2.1"
 
 [dependencies.i18n-embed]
-version = "0.15"
+version = "0.14"
 features = [
     "fluent-system",
     "desktop-requester",
 ]
 
 [dependencies.i18n-embed-fl]
-version = "0.9"
+version = "0.7"
 
 [dependencies.image]
-version = "0.25"
+version = "0.24"
 features = [
     "bmp",
     "dds",
-    "exr",
-    "ff",
+#    "exr",
+#    "ff",
     "gif",
     "hdr",
     "ico",
@@ -110,7 +110,7 @@
 version = "0.5"
 
 [dependencies.infer]
-version = "0.16"
+version = "0.16"
 
 [dependencies.itertools]
 version = "0.13"
@@ -157,11 +157,11 @@
 version = "1.10"
 
 [dependencies.rust-embed]
-version = "8.5"
+version = "8"
 features = ["debug-embed"]
 
 [dependencies.rusty-chromaprint]
-version = "0.2"
+version = "0.3"
 
 [dependencies.serde]
 version = "1.0"
@@ -190,7 +190,7 @@
 features = ["xxh3"]
 
 [dependencies.zip]
-version = "2.2"
+version = "0.6"
 features = [
     "aes-crypto",
     "bzip2",
@@ -208,10 +208,10 @@
     "dep:libheif-rs",
     "dep:libheif-sys",
 ]
-libavif = [
-    "image/avif-native",
-    "image/avif",
-]
+# libavif = [
+#     "image/avif-native",
+#     "image/avif",
+# ]
 libraw = ["dep:libraw-rs"]
 
 [target."cfg(windows)".dependencies.file-id]
